pkg/qemu/qemu_driver.go

```diff
@@ -191,6 +281,17 @@ func (l *LimaQemuDriver) removeVNCFiles() error {
 	return nil
 }
 
+func (l *LimaQemuDriver) killVhosts() error {
+	var errs []error
+	for i, vhost := range l.vhostCmds {
+		if err := vhost.Process.Kill(); err != nil && !errors.Is(err, os.ErrProcessDone) {
+			errs = append(errs, fmt.Errorf("Failed to kill virtiofsd instance #%d: %w", i, err))
+		}
+	}
+
+	return errors.Join(errs...)
```

Comment:
We have been using go-multierror: https://github.com/lima-vm/lima/blob/658ef436ee7d7bf7f1c48499058fb46aed3777e4/go.mod#L25

Eventually we can switch away from `errors.Join` though.